word_count.rb

```diff
@@ -0,0 +1,36 @@
+class Phrase
+
+  def initialize(phrase)
```

Comment:
do not perform any logic inside initialize method. just assign values to instance variables or default values to it.